src/test/scripts/applications/GAN/doc.md

```diff
@@ -0,0 +1,65 @@
+Simple GAN model:
```

Comment:
i would not include this file, since you specify in the actual files what is included.
but if you keep it, you need to add the license in the top.
see README.md for an example